src/jolt/driver/joltcb.py

```diff
@@ -118,12 +125,41 @@ def __str__(self):
         return self.args[1]
 
 class JOLTComputerBoard():
-    
     def __init__(self, simulated=False):
         self._ser_access = threading.Lock()
+        # Let's see if it actually raises
         self._serial = self._find_device(simulated=simulated)
         self.counter = 0
-        
+
+        # Version handling
+        be_hw_version = self.get_be_hw_version().lower()
+        if JOLT_V1_HW_IDENTIFIER in be_hw_version:
+            logging.info("JOLT V1 detected")
+            self.version = 2  # DEBUG
+        elif JOLT_V2_HW_IDENTIFIER in be_hw_version:
+            logging.info("JOLT V2 detected")
+            self.version = 2
+        else:
+            self.version = -1
+            logging.warning("Could not detect JOLT version")
```

Comment:
Please add the `be_hw_version` in the warning message, to make it easier to debug when this happens.